Snakefile

```diff
@@ -28,17 +37,17 @@ array_positions = (
     )
 )
 
-def nanopore_handling():
-    if config["sequence_type"] == "ont":
-        return {"maxd":500, "qual":10}
-    else:
-        return {"maxd":0,   "qual":50}
-
-def get_manual_tp_p(case, somatic):
-    if "models" in config:
-        if somatic in config["models"]:
-            return([config["models"][case][somatic]["tp"], config["models"][case][somatic]["ploidy"]])
-    return(["NA", "NA"])
+if "maxd" in config and "qual" in config:
+    pass
+elif "sequence_type" in config:
+    config["maxd"] = config["sequence_type_defaults"][config["sequence_type"]]["maxd"]
+    config["qual"] = config["sequence_type_defaults"][config["sequence_type"]]["qual"]
+else:
+    logger.warning(
+        f"No sequence_type or maxd/qual parameters.  Using 'short' defaults."
+    )
+    config["maxd"] = config["sequence_type_defaults"]["short"]["maxd"]
+    config["qual"] = config["sequence_type_defaults"]["short"]["qual"]
```

Comment:
Mostly change just because snakemake --lint suggests separating functions from rules.
And 'maxd' and 'qual' can just be specified in the config.

Reply:
The problem is that this change requires a user to know what the quality thresholds should be for illumina/ONT data, and means more config tooling, compared to a preset for either short or long reads. Adds end-user complexity for the sake of code neatness?

Reply:
Ah, I see the relevant lines in the config. Functionally the same with more control. Seems fine.